main.css

```diff
@@ -344,3 +346,356 @@
     color: black;
   }
 }
+
+@media only screen and (min-width: 768px){
+  body{
+    color: #091E42;
+  }
+
+  .header-section {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items:center;
+    background-color: #3e3c3c;
+    color: #fff;
+    padding:10px;
+    width: 100%;
+  }
+
+  .header-section .title {
+    text-decoration: none;
+    color: #fff;
+    margin-left: 20px;
+  }
+
+  .header-section .menu {
+    display: flex;
+    flex-direction: row;
+    margin-right: 20px;
+  }
+
+  .header-section .menu .menu-links {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .header-section .menu .menu-links a {
+    text-decoration: none;
+    list-style: none;
+    margin-right: 14px;
+    color: white;
+  }
+
+  .header-section .toggle-icon {
+    display: none;
+  }
+
+  .hero-section {
+    background: #1c1a19 url(./images/heroDesktopBg.png);
+    background-repeat: no-repeat;
+    background-size: 380px;
+    background-position-x: 930px;
+    background-position-y: -44px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    width: 100%;
+    height: 440px;
+    gap: 50px; 
+  }
+
+  .hero-section .hero-text {
+    color: #fff;
+    width: 400px;
+    height: 123px;
+  }
+  
+  .hero-section .hero-title h1 {
+    color: #f55800;
+    font-size: 72px;
+    font-family: 'Crete regular', serif;
+  }
+
+  .hero-section .hero-title h3 {
+    font-family: 'Inter', sans-serif;
+  }
+
+  .hero-section .hero-text .hero-desc {
+    font-family: 'Inter regular', sans-serif;
+    padding-top: 12px;
+    line-height: 20px;
+    font-size: 12px;
+  }
+
+  .hero-section .hero-social-links {
+    margin-left:16px;
+    list-style: none;
+    padding-top: 100px;
+  }
+
+  .hero-section .hero-social-links img {
+    width: 60%;
+    margin-top: 10px;
+  }
```

Comment:
- [ ] Hey, please kindly adjust the position of your social links by moving it up to the top a little so it aligns in the middle of your welcome section.

**YOUR DESIGN**

![contact](https://user-images.githubusercontent.com/59397159/167254998-098156b4-34ea-4f1c-9f38-d5c1f24f9474.png)


**FIGMA DESIGN**

![TRUE](https://user-images.githubusercontent.com/59397159/167255077-bc0fdb3a-cf2f-4c23-9bec-642989e8a2a3.png)

- Kindly make your image align with the end of the section and increase its width.

**YOUR DESIGN**

![IMAGE](https://user-images.githubusercontent.com/59397159/167255167-9447f572-ddb2-4e2f-91bd-9cb2d2e8f3e7.png)

**FIGMA DESIGN**

![HMM](https://user-images.githubusercontent.com/59397159/167255200-8daef903-7b1e-401c-987d-1d3971157735.png)